### ISSUE TITLE
Staff Profile sidenav accessibility defects - no label and empty list

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15519]
## Describe the defect
On Staff profile pages, there is a side nav in the left column and there are 2 accessibility issues:
1. This is in a `<nav>` element but is does not have a unique label to differentiate it from other navigations on the same page.
2. There is an empty `<ul>` element within the nav. Empty semantic elements create confusion for assistive tech users

## Questions/Notes
- Should this even be a `<nav>`? There is only 1 link in here - it's is used to navigate back to the leadership back but I'm not sure that should qualify it as an entire navigation landmark
- This doesn't show on small screens, if it's not important enough for mobile, which is what we should be designing for first, do we really need this on desktop?
- Having the heading in the left column, above the nav throws off the heading order on the page. Ideally this wouldn't be there, it could be text styled in this way but wouldn't be a semantic heading

## To Reproduce
1. Go to a Staff detail page, like [this one](https://www.va.gov/salem-health-care/staff-profiles/allen-r-moye/)
4. Inspect the sidenav
5. Notice that there is no `aria-label`
6. Also note that there is an empty `<ul>`

## Recommendations

- [ ] Decide what to do about the navigation
- [ ] Ensure that there are no empty tags (i.e. if there is a `<ul>` there are list items in it but if not, the `<ul>` does not appear in the DOM)

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0b604d00-6b64-41aa-9b8b-bef8ce2d68f4)

 ## Additional context
This is at least the 3rd iteration of a sidenav I've seen so far on the site, the others being on VAMC pages and Benefits Hubs. Should componentizing this be considered?

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by laflannery** [comment]
@mmiddaugh I have taken a closer look at this issue and there does seems to be an IA/UX consideration here that I think if we are going to touch this, we way want to not just band-aid fix it but look at it more overall. However, there are definitely some technical considerations here as well.

So, I think this comes down to:
- Do we want to/is it worth it to band-aid fix this for now so that the error gets cleared from AMP Then we can put in some more time/effort/resources into figuring out a more correct solution for this template overall?
- Or do we skip the band-aid and go directly to the longer term solution?

**Comment by xiongjaneg** [comment]
@xiongjaneg Discuss with Michelle, potential for Q1

**Comment by mmiddaugh** [comment]
Do we have any idea what level of effort is needed for the band-aid/stop gap measure, @xiongjaneg?

**Comment by xiongjaneg** [comment]
@mmiddaugh I'll check with Eli when they're back in tomorrow. It would likely be the second AC for the quick fix: Ensure that there are no empty tags (i.e. if there is a <ul> there are list items in it but if not, the <ul> does not appear in the DOM)

The long term fix would involve the first AC on figuring out the navigation solution.

**Comment by laflannery** [comment]
@xiongjaneg an `aria-label` needs to be added to the `<nav>` element to band-aid this. It's not in the current ACs because I wasn't sure exactly how we wanted to handle this. Without that label, the AMP error will not go away.

**Comment by laflannery** [comment]
Closing this because an epic isn't needed here - these issues are all contained within the a11y debt epic
